buildSrc/src/main/kotlin/ai.java-conventions.gradle.kts

```diff
@@ -150,6 +150,7 @@ dependencyCheck {
   skipConfigurations = listOf("errorprone", "spotbugs", "checkstyle", "annotationProcessor")
   failBuildOnCVSS = 0f // fail on any reported CVE
   suppressionFile = rootProject.file("buildscripts/dependency-check-suppressions.xml").absolutePath;
+  nvd.apiKey = System.getenv("NVD_API_KEY")
```

Comment:
@trask nvd is not recognized by CI. Is there some other setting I need to do? I've added key to github secrets.